.gitignore

```diff
@@ -1 +1,2 @@
-vulcanized.html
+bower_components
+.DS_Store
```

Comment:
Please refer to my previous comment on iron-toolbar here: https://github.com/PolymerElements/paper-toolbar/pull/1#discussion_r28701047

Let me know what you think!